Godeps/Godeps.json

```diff
@@ -1522,8 +1522,8 @@
 		},
 		{
 			"ImportPath": "github.com/evanphx/json-patch",
-			"Comment": "v3.0.0-23-g94e38aa",
```

Comment:
I'm surprised none of the files under the staging directories needed this update as well